### DISCUSSION TITLE
Update zoxide DB from browsing

### DISCUSSION THREAD

**Discussion by fdw** [sxyazi/yazi#860] [4. Ideas]
I love the built-in support for zoxide! However, would it also be possible to tell zoxide which directories I've opened, so that the DB there is automatically populated not just by `cd`ing there, but also from yazi? Or is that already possible and I've just missed it?

**Comment by sxyazi** [comment]
Hi, zoxide will be reimplemented as a plugin in the next version of Yazi, which will solve this problem

**Reply by ccoVeille** [reply]
Issue is here for anyone interested #865

**Comment by sxyazi** [comment]
Done by https://github.com/sxyazi/yazi/issues/865, with the latest main branch of Yazi, you can add:

```lua
require("zoxide"):setup {
	update_db = true,
}
```

to [your `init.lua` file ](https://yazi-rs.github.io/docs/plugins/overview) to enable it.
